server/helpers/loadConfig.js

```diff
@@ -0,0 +1,86 @@
+// config intended to be passed on to frontend
+// keys in process .env must be referencesd explicitly
+// so dotenv-webpack is able to include them in the test bundle for Mocha tests
+const check = require('check-types')
+
+// process.env.npm_package_version is undefined on Heroku
+const appVersion = require('../../package.json').version
+
+const {env} = process
+const booleans = ['true', 'false']
```

Comment:
I would directly declare a helper function "isBoolStr" and use that instead of repeatedly calling includes